test-cases/performance/perf-regression-throughput-baremetal-5gb.yaml

```diff
@@ -0,0 +1,63 @@
+test_duration: 580
+
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..312500",
+                    "cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=312501..625000",
+                    "cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=625001..937500",
+                    "cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=937501..1250000"]
+
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=1250000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..1250000"
+stress_cmd_r: "cassandra-stress read no-warmup  cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..1250000,625001,1250000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=40 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..1250000,625001,1250000)' "
+stress_multiplier: 1
+
+cluster_backend: baremetal
+user_credentials_path: '~/.ssh/scylla-qa-ec2'
+
+use_preinstalled_scylla: false
+run_db_node_benchmarks: false
+ssh_transport: fabric
+
+use_mgmt: false
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+db_nodes_public_ip:
```

Comment:
We need to see how this part of the configuration isn't coming from public source code